portal/server/controllers/utilisation-report-service/record-correction/provide-utilisation-report-correction/index.ts

```diff
@@ -27,7 +28,20 @@ export const getProvideUtilisationReportCorrection = async (req: GetProvideUtili
 
     const feeRecordCorrection = await api.getFeeRecordCorrection(userToken, bankId, correctionId);
 
-    const savedFormValues = await api.getFeeRecordCorrectionTransientFormData(userToken, bankId, correctionId);
+    let savedFormValues: GetFeeRecordCorrectionTransientFormDataResponseBody = {};
+
+    /**
+     * If the user is coming from a link on the Report GEF utilisation and fees page
+     * then we want to clear any saved form data for the correction as they are starting
+     * a new journey.
+     */
+    const isNewJourney = req.headers.referer?.includes('utilisation-report-upload');
```

Comment:
is it possible to double check this returns the 'utilisation-report-upload' locally?
Locally this did not return the last part of the url for me

Reply:
Yep I have tested this locally and it works correctly and is captured by and e2e test case also